src/Concerns/HasFlexible.php

```diff
@@ -49,6 +49,10 @@ public function cast($value, $layoutMapping = [])
      */
     public function toFlexible($value, $layoutMapping = [])
     {
+        if (is_a($value, 'Whitecube\NovaFlexibleContent\Layouts\Collection')) {
```

Comment:
I would prefer using the scalar `::class` classname representation here since it can be shortened with a `use` statement, making it more readable and enabling useful features in some code editors.